crates/bevy_internal/Cargo.toml

```diff
@@ -77,7 +77,7 @@ serialize = [
 multi-threaded = [
   "bevy_asset/multi-threaded",
   "bevy_ecs/multi-threaded",
-  "bevy_render/multi-threaded",
+  "bevy_render?/multi-threaded",
```

Comment:
Neat, I hadn't seen this syntax before. Do you have docs to link for this?

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html doesn't appear to mention it.

Reply:
Aha: https://doc.rust-lang.org/cargo/reference/features.html

> The "package-name/feature-name" syntax will also enable package-name if it is an optional dependency. Often this is not what you want. You can add a ? as in "package-name?/feature-name" which will only enable the given feature if something else enables the optional dependency.